src/components/Datatable/Pagination/PageButton.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import {
+  getColor,
+  getFontSize,
+  getFontWeight,
+  getLineHeight,
+  pxToRem,
+} from '../../../utils/helpers';
+import { Button } from '../../Button';
+import { ButtonVariants } from '../../Button/Button.enums';
+import { PageButtonProps } from './Pagination.types';
+
+const PageButton = styled(Button)
+  .attrs(() => ({
+    variant: ButtonVariants.text,
+  }))
+  .withConfig<PageButtonProps>({
+    shouldForwardProp: (prop) => prop !== 'isActive',
+  })`
+  display: inline-block;
+  text-align: center;
+  font-size: ${({ isActive }) =>
+    isActive ? getFontSize('lg') : getFontSize('md')};
+  font-weight: ${({ isActive }) =>
+    isActive ? getFontWeight('bold') : getFontWeight('regular')};
+  line-height: ${getLineHeight('lg')};
+  cursor: pointer;
+
+  color: ${getColor('graphite4B')} !important;
+  background-color: ${({ isActive }) =>
+    getColor(isActive ? 'graphiteH' : 'transparent')};
+
+  padding: ${pxToRem(6, 16)};
+  margin: ${pxToRem(0, 6)};
+  height: ${pxToRem(32)};
```

Comment:
Button width is changing based on content and then pagination is jumping. I would set fixed width.

Reply:
I kept it flexible because in some cases we can have high page numbers in thousands or tens of thousands in case of a million IPs

Reply:
I added a fixed width for spinner so no more horizontal jumping but there are still vertical jumps because of variation in the table cells' heights.